solr/packaging/test/test_start_solr.bats

```diff
@@ -31,6 +31,7 @@ teardown() {
 @test "SOLR11740 check f" {
   solr start
   solr start -p 7574
+  sleep 5
```

Comment:
You can assert solr is running, which will return status 0 immediately if solr is running, else wait.
```suggestion
  bin/solr assert --started http://localhost:8983/solr --timeout 5000
  bin/solr assert --started http://localhost:7574/solr --timeout 5000
```